impl/src/test/java/org/ehcache/impl/persistence/FileBasedStateRepositoryTest.java

```diff
@@ -16,17 +16,14 @@
 
 package org.ehcache.impl.persistence;
 
+import org.ehcache.impl.serialization.SerializerTestUtilities;
 import org.ehcache.impl.serialization.TransientStateHolder;
 import org.ehcache.spi.persistence.StateHolder;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.ObjectInputStream;
-import java.io.ObjectOutputStream;
+import java.io.*;
```

Comment:
I think your IDE is collapsing the imports to .*, you can disable that. Please undo this change.